.kitchen.yml

```diff
@@ -22,10 +14,31 @@ provisioner:
         - elasticsearch
 
 platforms:
+  # update base ubuntu image and install
+  # software-properties-common  for add-apt-repository
+  <% @ubuntu_provision_commands = [
```

Comment:
I think it may be best to move each OS-specific provision command to a bash script outside of this file.